File: src/test/resources/corpus/aluno/primo-testando-todos-os-divisores.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();

        int divisores = 0;
        for (int i = 1; i <= n; i++) {
            if (n % i == 0) {
                divisores++;
            }
        }

        if (divisores == 2) {
            System.out.println("PRIMO");
        } else {
            System.out.println("NAO PRIMO");
        }
    }
}

File: src/test/resources/corpus/c/aluno/primo-testando-todos-os-divisores.c
#include <stdio.h>

int main(void) {
    int n, i, ehPrimo;

    printf("Digite um numero: ");
    scanf("%d", &n);

    ehPrimo = 1;
    if (n < 2) {
        ehPrimo = 0;
    }

    // testa todos os divisores de 2 ate n-1
    for (i = 2; i < n; i++) {
        if (n % i == 0) {
            ehPrimo = 0;
        }
    }

    if (ehPrimo == 1) {
        printf("PRIMO\n");
    } else {
        printf("NAO PRIMO\n");
    }

    return 0;
}
